docs/en/engines/table-engines/integrations/mongodb.md

```diff
@@ -30,18 +30,30 @@ CREATE TABLE [IF NOT EXISTS] [db.]table_name
 
 -   `password` — User password.
 
+-   `options` — MongoDB connection string options (optional parameter).
```

Comment:
Do you think we need to add more options for MongoDB table engine? Probably just add trivial boolean use_ssl instead? I haven't found something close to this format in other table engines.

Reply:
There are many other options that can be passed here:

https://docs.mongodb.com/manual/reference/connection-string/#std-label-connections-connection-options